controllers/PostsController.js

```diff
@@ -22,22 +22,89 @@ class PostsController {
     }
   }
 
-  async createPost(req, res) {
-    const { categories, ...data } = req.body;
-    if (isNaN(Number(data.authorId))) {
-      return res.status(400).send("Wrong Type of authorId");
+  //sortBy: newestpost , popular(likes no.), newest comment
```

Comment:
What does "limit" mean?

Reply:
Requester can limit their result and use page as offset to get the limited data. It will help reduce data transfer and improve performance if FE also only render the usable data.